languagetool-language-modules/es/src/main/resources/org/languagetool/rules/es/grammar.xml

```diff
@@ -27095,6 +27114,11 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
         </rulegroup>
         <rulegroup id="MAMA" name="mama/mamá">
             <url>https://www.rae.es/dpd/mam%C3%A1</url>
+            <antipattern>
+                <token regexp="yes">de|en|sobre|por|para|con|sin</token>
+                <token regexp="yes">la|las|el|los|su|sus|una|unas</token>
+                <token regexp="yes">mama|mamas</token>
+            </antipattern>
```

Comment:
This will cause some FNs:
"de la mama" -> "de la mamá" (madre)
I know that "mama" (=madre) is also possible. But there are users that complain if we don't suggest "mamá".